src/rules/noAnyRule.ts

```diff
@@ -42,7 +43,10 @@ export class Rule extends Lint.Rules.AbstractRule {
 
 class NoAnyWalker extends Lint.RuleWalker {
     public visitAnyKeyword(node: ts.Node) {
-        this.addFailureAtNode(node, Rule.FAILURE_STRING);
+        const fix = this.createFix(
+            this.createReplacement(node.getStart(), node.getWidth(), "{}"),
```

Comment:
What is the benefit of replacing `any` with `{}`? I don't think this rule can be autofixed since the point is to use a specific type.